releases/release_notes_09_2022.md

```diff
@@ -1,59 +1,84 @@
-# About 
+# About
 Welcome :-)
 
 We are thrilled to share with you Confidential Containers (COCO) release X.X.X .
 
-In this document we will take you through the content of this release, installation instructions, deploying workloads and troubleshooting if things go wrong. 
+In this document we will take you through the content of this release, installation instructions, deploying workloads and troubleshooting if things go wrong.
 
-# Release notes 
+# Release notes
 
-## Goals 
+## Goals
 
 This release focused on the following:
 
-- **Simplicity** - Using the operator to deploy and configure
-- **Stability** - Supporting CI for the key workflows of the release
+- **Simplicity** - Using a dedicated Kubernetes operator, the COCO operator, to deploy and configure
+- **Stability** - Supporting Continuous Integration (CI) for the key workflows of the release
 - **Documentation** - Details instruction of how to deploy and use this release
 
-## Use cases 
+## Use cases
 
 This release supports the following use cases:
 
 - Creating a sample COCO workload
 - Creating a COCO workload using a pre-existing encrypted image
-- Creating a COCO workload using a pre-existing encrypted image on CC HW
+- Creating a COCO workload using a pre-existing encrypted image on hardware with support for Confidential Computing (CC HW)
 - Building a new encrypted container image and deploying it as a COCO workload
 
-## Limitations 
-
-The following are known limitations of this release: 
-
-- A
-- B
-- C
-
-# Installing 
-
-The COCO solution can be installed, uninstalled and configured using the COCO operator. 
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * AMD SEV is tested by the CI (with some limitations regarding attestation, see below)
+  * Intel TDX is expected to work, but not currently tested in the CI
+  * S390x is not supported by the COCO operator
+- Attestation and key brokering support is still under development
+  * THe disk-based key broker client (KBC) is still the primary method used for development, even if it will never be an acceptable approach in productino.
```

Comment:
`productino` -> `production`

Reply:
Typo fixed, thanks. The changes we made in person should be there. What is missing in your opinion?